commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/internal/search/test/CommercePriceListIndexerTest.java

```diff
@@ -132,6 +138,8 @@ public void testSearch() throws Exception {
 	private CommercePriceListLocalService _commercePriceListLocalService;
 
 	@DeleteAfterTestRun
+	private Company _company;
+
 	private Group _group;
```

Comment:
@luca-pellizzon don't we still want to delete `_group` after the test run?

Reply:
we are removing the whole company